components/sbm-recipes-mule-to-boot/src/main/java/org/springframework/sbm/mule/actions/JavaDSLAction2.java

```diff
@@ -228,4 +237,25 @@ private SpringBootApplicationProperties findOrCreateDefaultApplicationProperties
                 .findFirst()
                 .get();
     }
+
+    public void setMuleTriggerMeshTransformEnabled(boolean mode) {
```

Comment:
Could you replace this with a @Setter lombok annotation, just to be  consistent, please